libs/ddd-angular/src/generators/util/generator.ts

```diff
@@ -9,4 +10,5 @@ export default async function (tree: Tree, options: DddUtilGeneratorSchema) {
     shared: true,
     standalone: false,
   });
+  removeFiles(tree, `shared-util-${sanitize(options.name)}`);
```

Comment:
Could we move the sanitize logic within the removeFiles function please :rocket: